crates/oxc_sourcemap/tests/fixtures/esbuild/visualizer.snap

```diff
@@ -0,0 +1,208 @@
+---
+source: crates/oxc_sourcemap/tests/main.rs
+input_file: crates/oxc_sourcemap/tests/fixtures/esbuild/example.js
+snapshot_kind: text
+---
+- index.tsx
+(0:0) "import { " --> (1:0) "import { "
+(0:9) "h, " --> (1:9) "h as "
+(0:9) "h, " --> (1:14) "u, "
```

Comment:
This duplicate mapping is working similar to evanw.

![image](https://github.com/user-attachments/assets/2e14837a-fb7f-4216-937e-f870d435c38f)